builder/fixtures/buildpacks/unix/has-buildpack-config/bin/compile

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# vim: set ft=sh
```

Comment:
Probably should remove this `vim: set ft=sh` bit

Reply:
Fixed